daemon/container_linux.go

```diff
@@ -297,8 +297,8 @@ func populateCommand(c *Container, env []string) error {
 		Resources:          resources,
 		AllowedDevices:     allowedDevices,
 		AutoCreatedDevices: autoCreatedDevices,
-		CapAdd:             c.hostConfig.CapAdd,
-		CapDrop:            c.hostConfig.CapDrop,
+		CapAdd:             c.hostConfig.CapAdd.Slice(),
```

Comment:
reason behind not changing the type in `execdriver.Command` is because I didn't want to modify it since it will eventually be deleted along whole execdriver, I can change it tho if you prefer